src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/AspNetCoreInstrumentationEventSource.cs

```diff
@@ -57,12 +58,14 @@ public void RequestIsFilteredOut(string handlerName, string eventName, string op
             this.WriteEvent(2, handlerName, eventName, operationName);
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode", Justification = "Parameters to this method are primitive and are trimmer safe.")]
```

Comment:
Suppress the warnings for the other two events in this class as well?

Reply:
This will go away with https://github.com/dotnet/runtime/pull/83751, but only when targeting .NET 8, so not any time soon for this repo.